public/_data.json

```diff
@@ -31,21 +31,5 @@
 
   "presskit": {
     "title": "Press Kit"
-  },
```

Comment:
Originally we believed these would be separate pages, eventually we consolidated everything. So these pages are no longer needed.